src/modelvalidityguard.cpp

```diff
@@ -64,5 +66,6 @@ void ModelValidityGuard::validate()
             return;
     }
 
+    ParsedDataStorage::instance()->resetProjectBuildersCount();
```

Comment:
I'm not sure if resetting those indexes at any point is 100% safe. Is it really needed? Not resetting seems as more safe (as long as there are less than 2G updates in a single run ;) )

Reply:
I just wanted to avoid the situation in which potential counter overflowing is not handled. I believe that resetting counters at this point is safe, as it is called when no processing of any files is taking place at the moment. However, if code simplicity suffers from this solution, than it should be removed IMO.

Reply:
ok, I understand, keep it as is

but about overflow: we do not send parsing events more often than what, 1 per second? that means that int is enough for ... >60years? :D

still - handling overflow is a nice thing ;)